From 446b19807781d73f214f959a8f4dab7662eed337 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Wed, 25 Mar 2020 18:18:45 +0000
Subject: [PATCH] drm/vc4: Enable audio on Pi4.

This could be a revert of "drm/vc4: hdmi: Add an audio support flag"
as it is no longer needed.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 drivers/gpu/drm/vc4/vc4_hdmi.c | 2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/gpu/drm/vc4/vc4_hdmi.c
+++ b/drivers/gpu/drm/vc4/vc4_hdmi.c
@@ -1751,6 +1751,7 @@ static const struct vc4_hdmi_variant bcm
 
 static const struct vc4_hdmi_variant bcm2711_hdmi0_variant = {
 	.id			= 0,
+	.audio_available	= true,
 	.max_pixel_clock	= 297000000,
 	.registers		= vc5_hdmi_hdmi0_fields,
 	.num_registers		= ARRAY_SIZE(vc5_hdmi_hdmi0_fields),
@@ -1774,6 +1775,7 @@ static const struct vc4_hdmi_variant bcm
 
 static const struct vc4_hdmi_variant bcm2711_hdmi1_variant = {
 	.id			= 1,
+	.audio_available	= true,
 	.max_pixel_clock	= 297000000,
 	.registers		= vc5_hdmi_hdmi1_fields,
 	.num_registers		= ARRAY_SIZE(vc5_hdmi_hdmi1_fields),
